Source/Tests/LinuxSyscalls/EmulatedFiles/EmulatedFiles.h

```diff
@@ -30,7 +30,7 @@ namespace FEX::EmulatedFile {
       std::once_flag cpu_info_initialized{};
       std::string cpu_info{};
       using FDReadStringFunc = std::function<int32_t(FEXCore::Context::Context *ctx, int32_t fd, const char *pathname, int32_t flags, mode_t mode)>;
-      std::unordered_map<std::string, FDReadStringFunc> FDReadCreators;
+      std::unordered_map<std::string_view, FDReadStringFunc> FDReadCreators;
```

Comment:
There is one dynamic key (`procAuxv`) used with this, so a `string_view` creates a dangling reference on that one.

Reply:
Oop, you're right. dropped back to `std::string`